mycroft/skills/intent/__init__.py

```diff
@@ -57,6 +104,9 @@ def handle_utterance(self, message):
             reply = message.reply(
                 best_intent.get('intent_type'), best_intent)
             self.emitter.emit(reply)
+            # best intent detected -> update called skills dict
+            name = self.intent_to_skill[best_intent['intent_type']]
```

Comment:
Your variable names are misleading.  The intent_to_skill is more like "intent_to_skill_id"; and name is "skill_id".  It _works_, but will confuse those trying to figure out the code later.  Be kind to future generations.  :)